front/app/components/admin/ReportExportMenu/index.tsx

```diff
@@ -215,6 +201,28 @@ const ReportExportMenu = ({
     setDropdownOpened(value || !dropdownOpened);
   };
 
+  const downloadXlsxData = (data: XlsxData, fileName: string) => {
+    const workbook = XLSX.utils.book_new();
+
+    Object.entries(data).forEach(([sheet_name, sheet_data]) => {
+      // Added after production bug.
+      // Error from XLSX package: sheet names can't contain characters from the following array:
+      // [':', ']', '[', '*', '?', '/', '\\']
+      const sheetNameWithoutForbiddenChars = sheet_name.replace(
+        /[:\]\[*?/\\]/g,
+        '_'
+      );
```

Comment:
The problem was that in German some sheets used the * as it's used to distinguish between gender versions of a word (e.g. Besucher*innen).